tests/transforms/test_video_utils.py

```diff
@@ -60,7 +61,10 @@ def test_mp4_conversion(video_fixture, tmp_path):
 
     fps = 30
 
-    transformations.transform_to_mp4(video=video_fixture,
+    normalized_video = normalize_array(
+            video_fixture, 0, video_fixture.max())
```

Comment:
I would create a separate test called `test_mp4_conversion_with_normalized_video`. Since the current test if it failed, you wouldn't be able to **easily** tell of `normalize_array` failed or `transformations.transform_to_mp4` failed.

Reply:
looking more closely, this test does not need to normalize the video, as I had removed video normalization from `transform_to_mp4` but, now that I am looking at it more closely, I think this test needs a little work.

Reply:
reworked this test in https://github.com/AllenInstitute/segmentation-labeling-app/pull/54/commits/240d1df4e08a2a79c8c6dc13ee0b94d588754c4a